### PR TITLE
fix: helm chart folder name should be same as the chart name

### PR DESCRIPTION
**Pull request by HarikrishnanBalagopal**
Signed-off-by: Harikrishnan Balagopal <harikrishmenon@gmail.com>

### CODE REVIEW

File: internal/transformer/k8stransformer.go [hunk 1]
```diff
@@ -218,9 +214,9 @@ func (kt *K8sTransformer) WriteObjects(outputPath string, transformPaths []strin
 	return nil
 }
 
-func (kt *K8sTransformer) generateHelmMetadata(outputPath string, values outputtypes.HelmValues) error {
+func (kt *K8sTransformer) generateHelmArtifacts(outputPath string, values outputtypes.HelmValues, transformPaths []string) error {
 	deployPath := filepath.Join(outputPath, common.DeployDir)
-	helmPath := filepath.Join(deployPath, "helm")
+	helmPath := filepath.Join(deployPath, "helm", kt.Name)
```
Comment (ashokponkumar) [inline internal/transformer/k8stransformer.go:219]: Lets call it "helm-charts". Probably we can add a helm chart for the ci/cd artifacts too.

**Review by ashokponkumar — CHANGES_REQUESTED** [review]
Also, What is the folder structure for kustomize artifacts? is it good to have the folder name there too?

Should we put ci/cd inside each of these yamls, kustomize and helm too, so that it allows for install of ci/cd using them too?

**Comment by HarikrishnanBalagopal** [comment]
> 
> 
> Also, What is the folder structure for kustomize artifacts? is it good to have the folder name there too?
> 
> Should we put ci/cd inside each of these yamls, kustomize and helm too, so that it allows for install of ci/cd using them too?

https://github.com/konveyor/move2kube/blob/main/internal/transformer/k8stransformer.go#L160-L176

File: internal/transformer/k8stransformer.go (hunk 1 above)
Comment (HarikrishnanBalagopal) [reply internal/transformer/k8stransformer.go:219]: We already have CI/CD don't we?

Comment (HarikrishnanBalagopal) [reply internal/transformer/k8stransformer.go]: `Lets call it "helm-charts"` done

**Review by ashokponkumar — APPROVED** [review]

**Pull request merged**